examples/vision/cutmix.py

```diff
@@ -0,0 +1,381 @@
+"""
+Title: CutMix data augmentation for image classification
+Author: [Sayan Nath](https://twitter.com/SayanNa20204009)
+Date created: 2021/03/25
+Last modified: 2021/03/25
+Description: Data augmentation with CutMix for image classification on CIFAR-10.
+"""
+
+"""
+## Introduction
+"""
+
+"""
+_CutMix_ is a data augmentation technique that addresses the issue of
+information loss and inefficiency present in regional dropout strategies. 
+Instead of removing pixels and filling them with black or grey pixels or
+Gaussian noise, you replace the removed regions with a patch from another image,
+while the ground truth labels are mixed proportionally to the number of pixels
+of combined images. CutMix was proposed in
+[CutMix: Regularization Strategy to Train Strong Classifiers with Localizable Features](https://arxiv.org/pdf/1905.04899.pdf)
+(Yun et al., 2019).
+
+CutMix Augmentation is used when there is presence of uninformative pixel during training.
```

Comment:
Suggestion:  the last sentence ("...uninformative pixel[s]...") can be included in the main paragraph for "coherency" + we can use some of the stuff from the paper

For example:

```suggestion
_CutMix_ is a data augmentation technique that addresses the issue of
information loss and inefficiency present in regional dropout strategies. 
Instead of removing pixels and filling them with black or grey pixels or
Gaussian noise, you replace the removed regions with a patch from another image,
while the ground truth labels are mixed proportionally to the number of pixels
of combined images. The CutMix technique can make efficient use of training
pixels, while retaining the regularization effect of regional dropout.

This data augmentation technique was proposed in
[CutMix: Regularization Strategy to Train Strong Classifiers with Localizable Features](https://arxiv.org/pdf/1905.04899.pdf)
(Yun et al., 2019).
```

LMKWYT